[RFR] Change the webserver default port

Luwangel opened the pull request:
- [x] Change the makefile to allow the PORT env variable
- [x] Change the webserver script to use the port argument

Luwangel committed:
Allow the user to change the default port

jpetitcolas commented on src/main/main-server.go:
```diff
@@ -2,12 +2,24 @@ package main
 
+func getPort() int {
+	var port int
+	flag.IntVar(&port, "port", DEFAULT_WEBSERVER_PORT, "an int")
```
s/an int/Port on which application should listen/

jpetitcolas commented on src/main/main-server.go:
```diff
@@ -2,12 +2,24 @@ package main
+	var port int
+	flag.IntVar(&port, "port", DEFAULT_WEBSERVER_PORT, "an int")
+	flag.Parse()
+	if port > 2000 && port < 3000 {
```
Weird condition. I would remove it, unless you can justify it .:)

sedy-bot committed:
Typo fix s/an int/Port on which application should listen/

As requested by jpetitcolas at https://github.com/marmelab/15-puzzle-go/pull/18#discussion_r154039523

Luwangel committed:
Remove the weird condition for the port
